alpaca/rest.go

```diff
@@ -469,6 +471,60 @@ func (c *client) ListOrders(status *string, until *time.Time, limit *int, nested
 	return orders, nil
 }
 
+// ListOrdersV2 returns the list of orders for an account,
+// filtered by the input parameters.
+func (c *client) ListOrdersV2(req ListOrdersRequest) ([]Order, error) {
+	urlString := fmt.Sprintf("%s/%s/orders", c.opts.BaseURL, apiVersion)
+	if req.Nested != nil {
```

Comment:
[minor] Why is Nested handled differently than all the other parameters?

Reply:
This is a relict from the other ListOrders function. I fixed it so nested is handled the same as all the other parameters.